### PR TITLE
R4 Observation Labs

### PR DESCRIPTION
**Pull request by Manjushavrk**
Description
----
Updated Performer.extension bindings and examples.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.
Screen Shot 2020-11-06 at 3 51 06 PM
Screen Shot 2020-11-06 at 3 56 01 PM
Screen Shot 2020-11-06 at 4 04 21 PM

### CODE REVIEW

File: lib/resources/r4/observation.yaml [hunk 1]
```diff
@@ -212,12 +223,42 @@ fields:
...
+              "url": "http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+            }
+          ],
+          "reference": "Practitioner/2797943"
+        }
   note: |
     <ul>
-       <li> Only `LA` extensions is supported </li>
-       <li> Performer extensions are supported for Vital Signs, but not Laboratory. </li>
+       <li> Vital Signs supports only `LA` extensions. </li>
```
Comment (BS062721) [inline lib/resources/r4/observation.yaml:258]: nit: extensions -> extension

Comment (Manjushavrk) [reply lib/resources/r4/observation.yaml]: updated: https://github.com/cerner/fhir.cerner.com/commit/f6b9c6c65ff58adf0865dda2b47dd98b65555a71#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcL258

File: lib/resources/r4/observation.yaml [hunk 2]
```diff
@@ -193,7 +193,18 @@ fields:
...
+  action:
+    - terminology
+    - create
+  binding:
+    description: Codes identifying the function of the performer.
+    terminology:
+    - display: Performer function codes
+      system: http://terminology.hl7.org/CodeSystem/v3-ParticipationType
+      info_link: http://hl7.org/fhir/R4/valueset-performer-function.html
+    - display: Participation codes
```
Comment (BS062721) [inline lib/resources/r4/observation.yaml:205]: I would like Fran's thought on display names for this links

**Review by BS062721 — APPROVED** [review]

File: lib/resources/r4/observation.yaml [hunk 3]
```diff
@@ -212,12 +223,42 @@ fields:
...
+                        ],
+                "text": "primary performer"
+              },
+              "url": "http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+            }
+          ],
+          "reference": "Practitioner/2797943"
+        }
   note: |
     <ul>
```
Comment (jamvar) [inline lib/resources/r4/observation.yaml:257]: @BS062721 unrelated to this PR for the performer field the example is,
```
{
   "reference": 'Practitioner/12724064'
 }
```
Shouldn't it be in `performer` field inside an array like, 
```
{
    "performer": [
        {
            "reference": 'Practitioner/12724064'
        }
    ]
}
```

File: lib/resources/r4/observation.yaml (hunk 2 above)
Comment (fpivonka) [reply lib/resources/r4/observation.yaml:205]: I think that was the name in the code set when you open that system link below. I'm okay with Participation Codes.

File: lib/resources/r4/observation.yaml [hunk 4]
```diff
@@ -212,12 +223,42 @@ fields:
...
+                          "display": "primary performer"
+                        }
+                        ],
+                "text": "primary performer"
+              },
+              "url": "http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+            }
+          ],
+          "reference": "Practitioner/2797943"
+        }
```
[the remark this answers is not in the document]
Comment (Manjushavrk) [reply lib/resources/r4/observation.yaml]: updated : https://github.com/cerner/fhir.cerner.com/commit/b3879fcb6a05b33d21207c3f86bec201b704cc01#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR227

File: lib/resources/r4/observation.yaml (hunk 3 above)
Comment (Manjushavrk) [reply lib/resources/r4/observation.yaml:263]: updated: https://github.com/cerner/fhir.cerner.com/commit/1d7546c21c9e25b6edbc48ae90cb38c95ee734b5#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR182-R187

**Review by fpivonka — APPROVED** [review]

File: lib/resources/r4/observation.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (jamvar) [reply lib/resources/r4/observation.yaml]: sorry i didn't removed it earlier. `reference` is part of `performer` remove the reference.

[the remark this answers is not in the document]
Comment (Manjushavrk) [reply lib/resources/r4/observation.yaml]: updated: https://github.com/cerner/fhir.cerner.com/commit/18a456686bef16a5f15af49c57a9ba05ca13b364#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcL259-L260

**Review by jamvar — APPROVED** [review]

**Comment by Mohammad-Isyroqi-Fathan** [comment]
Screenshots After Published:
Screen Shot 2020-11-09 at 6 13 36 PM
Screen Shot 2020-11-09 at 6 14 02 PM
Screen Shot 2020-11-09 at 6 14 09 PM

**Pull request merged**